### PR TITLE
update transformers to 4.20.1

### PR DESCRIPTION
**Pull request by FindHao**
- update transformers to 4.20.1
- remove patches

### CODE REVIEW

**Review by xuzhao9 — CHANGES_REQUESTED** [review]
We also need to update the patch function to fix the CI test failure

File: torchbenchmark/util/framework/huggingface/patch_hf.py [hunk 1]
```diff
@@ -16,16 +16,17 @@ def cache_model(name: str):
...
-        except subprocess.SubprocessError as e:
-            output_str = str(e.output)
-            if "previously applied" in output_str:
-                return
-            else:
-                print(str(output_str))
-                sys.exit(1)
+    pass
+    # import transformers
+    # transformers_dir = os.path.dirname(transformers.__file__)
```
Comment (xuzhao9) [inline torchbenchmark/util/framework/huggingface/patch_hf.py:21]: I guess there is a better way than just commenting the code:
1. detect if PATCH_DIR exists, if not, directly return
2. check if `patch_file_fullpatch` is ending with `.patch`, if not, skip.

**Review by xuzhao9 — CHANGES_REQUESTED** [review]

File: torchbenchmark/util/framework/huggingface/patch_hf.py (hunk 1 above)
Comment (FindHao) [reply torchbenchmark/util/framework/huggingface/patch_hf.py]: solved by https://github.com/pytorch/benchmark/pull/1025/commits/33196c26e7eeb2c1529aca1c608d7c7bc971033f.

**Review by xuzhao9 — APPROVED** [review]
LGTM. Thanks!

**Pull request closed without merging**